apis/go/mlops/scheduler/scheduler_grpc.pb.go

```diff
@@ -1,19 +1,3 @@
-/*
-Copyright 2022 Seldon Technologies Ltd.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
```

Comment:
re-instate license. we should add a ticket that at least makes sure that all files have a license and idealy adds the license to the files that are autogenerated.